frontend/pages/project-developers/new.tsx

```diff
@@ -508,7 +525,12 @@ const ProjectDeveloper: PageComponent<ProjectDeveloperProps, NakedPageLayoutProp
                     <span className="mr-2.5">{title}</span>
                     <FieldInfo infoText={infoText || getItemsInfoText(items)} />
                   </legend>
-                  <TagGroup name={name} setValue={setValue}>
+                  <TagGroup
+                    name={name}
+                    setValue={setValue}
+                    setValueOptions={{ shouldValidate: false }}
+                    // ShouldValidate disabled for not forcing the premature validation of the other fields of the page
```

Comment:
There's a discussion with @SARodrigues in #91. I don't think your PR should care about this, we will address it elsewhere.
```suggestion
```

Reply:
Ok!